drivers/sensor/fxos8700/fxos8700.c

```diff
@@ -18,6 +18,130 @@ LOG_MODULE_REGISTER(FXOS8700, CONFIG_SENSOR_LOG_LEVEL);
 /* Convert the range (8g, 4g, 2g) to the encoded FS register field value */
 #define RANGE2FS(x) (__builtin_ctz(x) - 1)
 
+#if DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
+#define DIR_READ(a)  ((a) & 0x7f)
+#define DIR_WRITE(a) ((a) | (1 << 7))
+#define ADDR_7(a) ((a) & (1 << 7))
+
+int fxos8700_transceive(const struct device *dev,
```

Comment:
Please split the driver changes into a standalone commit

Reply:
We have created one commit for fxos8700 sensor driver and its corresponding dts node, 
and similarly another commit for fxas21002 sensor driver and its respective dts node.

What change do you expect in this commit method ?

Reply:
Changed as per your suggestion